src/FffGcodeWriter.cpp

```diff
@@ -1528,6 +1529,16 @@ bool FffGcodeWriter::processSkinPart(const SliceDataStorage& storage, LayerPlan&
     return added_something;
 }
 
+bool FffGcodeWriter::processSanding(const SliceMeshStorage& mesh, const SliceLayerPart& part, const GCodePathConfig& line_config, LayerPlan& gcode_layer) const
```

Comment:
Yes, originally it was called sanding. At the end I renamed it to ironing but only renamed the user-visible strings.

I've now renamed the implementation under water as well, for consistency: https://github.com/Ultimaker/CuraEngine/commit/d927f7317a2601eb90f650e936377d3f7aa5b360